arelle/ModelRenderingObject.py

```diff
@@ -283,6 +296,18 @@ def strctMdlEffectiveChildNodes(self):
 
     def setHasOpenNode(self):
         self.hasOpenNode = True
+        
+    @property
+    def tagSelectors(self):
+        if isinstance(self.strctMdlParentNode, StrctMdlStructuralNode):
+            return self.strctMdlParentNode.tagSelectors
+        else:
+            return set()
+    def inheritedAspectValue(self, *args):
+        if isinstance(self.strctMdlParentNode, StrctMdlStructuralNode):
```

Comment:
Nit: `else` is unneeded because of short circuit return from the `if`. Might just use a ternary though